cluster-bootstrap/src/main/resources/reference.conf

```diff
@@ -99,9 +99,14 @@ akka.management {
     # Configured how we communicate with the contact point once it is discovered
     contact-point {
 
+      # The probe method. Valid values are akka-management and remoting. If akka-management, will use akka-management
+      # HTTP interface to discover seed nodes. If remoting, will use Akka remoting to discover seed nodes.
+      probe-method = "akka-management"
```

Comment:
why do we want to support both methods?

Reply:
You won't be able to do a rolling upgrade to the new method if you don't support both concurrently.